src/ast/data_type.rs

```diff
@@ -319,6 +319,10 @@ pub enum DataType {
     /// [`SQLiteDialect`](crate::dialect::SQLiteDialect), from statements such
     /// as `CREATE TABLE t1 (a)`.
     Unspecified,
+    /// Trigger data type, returned by functions associated with triggers
+    ///
+    /// [postgresql]: https://www.postgresql.org/docs/current/plpgsql-trigger.html
+    Trigger,
```

Comment:
for the datatype can we add a test case (noticed I couldn't find test code exercising it)?

Reply:
There is one here: https://github.com/LucaCappelletti94/sqlparser-rs/blob/7ab8fdb05ba937b0e14c376866b5ac280e5b59c6/tests/sqlparser_postgres.rs#L4866

Reply:
Ah right, didn't notice it was the datatype within a function, sounds good!